-code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-34220/23509

ERROR: Build errors found during clang-tidy run.
```
DQMServices/Core/src/DQMStore.cc:653:33: error: expected unqualified-id [clang-diagnostic-error]
          if (name.find(*ipath->) != std::string::npos) {
                                ^
Suppressed 1365 warnings (1364 in non-user code, 1 with check filters).
--
gmake: *** [config/SCRAM/GMake/Makefile.coderules:129: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/DQMServices/Core/python/DQMStore_cfi.py
+++ b/DQMServices/Core/python/DQMStore_cfi.py
@@ -5,10 +5,10 @@ DQMStore = cms.Service("DQMStore",
     verbose = cms.untracked.int32(0),
     # similar to LSBasedMode but for offline. Explicitly sets LumiFLag on all
     # MEs/modules that allow it (canSaveByLumi)
-    saveByLumi = cms.untracked.bool(True),
+    saveByLumi = cms.untracked.bool(False),
     #Following list has no effect if saveByLumi is False
     MEsToSave = cms.untracked.vstring(nanoDQMIO_perLSoutput.MEsToSave),  
-    trackME = cms.untracked.string("Muons/MuonRecoAnalyzer/Res_TkSta_theta"),
+    trackME = cms.untracked.string(""),
 
     # UNUSED: historical HLT configs expect this option to be present, so it
     # remains here, even though the DQMStore does not use it any more.

--- a/DQMServices/Core/src/DQMStore.cc
+++ b/DQMServices/Core/src/DQMStore.cc
@@ -650,7 +650,7 @@ namespace dqm::implementation {
              ipath != store_->MEsToSave_.end();
              ++ipath) {
           std::string name = (*it)->getFullname();
-          if (name.find(*ipath->) != std::string::npos) {
+          if (name.find(*ipath) != std::string::npos) {
             saveIt = true;
             //std::cout<<name<<" compared to"<<ipath->data()<<std::endl;
             break;

